SOPT-server-30/Assignment/assignment1/level2.ts

```diff
@@ -0,0 +1,39 @@
+import { Member } from './interface/Member';
+import { Dinner } from './interface/Dinner';
+
+const dinner: Dinner = {
+  member: [
+    {
+      name: "채정아",
+      group: "ob",
+    },
+    {
+      name: "김루희",
+      group: "yb",
+    },
+    {
+      name: "장서현",
+      group: "yb",
+    },
+    {
+      name: "이승헌",
+      group: "ob",
+    },
+    {
+      name: "남지윤",
+      group: "ob",
+    },
+  ],
+  shuffle(array) {
+    array.sort(() => Math.random() - 0.5);
+    return array;
+  },
+  organize(array) {
+    let members = this.shuffle(array);
```

Comment:
재할당을 하는 경우가 아니니
let 말고 const로 선언하는것은 어떨까요 ?!

```suggestion
    const members = this.shuffle(array);
```